### PR TITLE
fix docs

### PR DESCRIPTION
**Pull request by ilovesoup**
Add link to TiSparkR from userguide.

---
This change is [Reviewable](https://reviewable.io/reviews/pingcap/tispark/77)

### CODE REVIEW

**Comment by ilovesoup** [comment]
@Novemser PTAL

File: docs/userguide.md [hunk 1]
```diff
@@ -239,6 +239,10 @@ select count(*) from account;
 1 row selected (1.97 seconds)
 ```
 
+## TiSparkR
+A thin layer build for supporting R language with TiSpark
```
Comment (lilin90) [inline docs/userguide.md]: It'd be better to make it a complete sentence: "TiSparkR is a thin layer build for supporting the R language with TiSpark."

File: docs/userguide.md [hunk 2]
```diff
@@ -239,6 +239,10 @@ select count(*) from account;
...
+Please refer to [this document](../TiSparkR/README.md) for usage.
```
Comment (lilin90) [inline docs/userguide.md]: The "please" here is overdoing the politeness. Generally do not use "please" in user guide. Reference: [https://developers.google.cn/style/tone#politeness](url)

Comment (ilovesoup) [reply docs/userguide.md]: both done.

**Pull request merged**